00-open-a-shapefile.Rmd

```diff
@@ -78,237 +75,254 @@ Working With Shapefile Attributes in R </a>
-{: .notice }
-
-``` {r Shapefile-attributes}
-#view attributes of the layer
-aoiBoundary
-```
+Note: The acronym, `OGR` refers to the OpenGIS Simple Features Reference 
+Implementation. 
+<a href="https://trac.osgeo.org/gdal/wiki/FAQGeneral" target="_blank"> 
+Learn more about OGR.</a> {: .notice1}  
 
-Here `R` prints a number of attributes including information about the object's `coordinate reference system (CRS)`. Note that the objects coordinates are in the projection: UTM zone 18N. This information is critical to interpreting the object extent printed on the previous line. Also, note that our object only contains a single feature. We will discuss metadata and attributes in more
-detail a bit later. Finally, we see that there is one variable named "id" with min and max values of 1.
+#Shapefile Metadata & Attributes
 
-We can look at all of the associated data attributes by printing the contents of the `data` slot with `@data`.  
+When we imported the `HarClip_UTMZ18` shapefile layer into `R` (as our
+`aoiBoundary_HARV` object), the `readOGR()` function automatically stored
+informatation about and associated with the data. We are particularly interested
+in the geospatial *metadata* that contains information on the data set and
+*attributes* that contain non-geographic information about the items in the data
+set.  
 
-```{r Shapefile-attributes-2}
-#just view the attributes of the data
-(aoiBoundary@data)
-```
+Below, we discuss some of the most commonly used metadata and attribute data
+used with spatial data.  The
+[Shapefile Metadata & Attributes in R]({{site.baseurl}}R/shapefile-attributes-in-R/) 
+lesson provides more information on both metadata and attributes as well as
+using attributes to subset and plot data.  
 
-Recall from the previous step that our object only has a single feature. After printing the data attributes we see that the _id_ of this feature is 1. The 0 printed in the first column is simply a `row.name`. We can test this by issuing the following command to the `R` console: `row.names(aoiBoundary@data)`.
+##Spatial Metadata
+Key metadata for all shapefiles include:
 
-Now that we have become familiar with our object attributes we can visualize our data. The following code chunk will produce a box representing the boundaries specified in the shapefile that we just imported.
+1. **Object Type:** - what is the class of the imported object. 
+2. **Coordinate Reference System (`CRS`)** - what projection is the data in.
+3. **Extent** - the spatial extent of the shapefile. Note that the spatial
+extent for a shapefile represents the extent for ALL polygons (or spatial
+objects) in the shapefile!
 
-``` {r Plot-shapefile}
-#create a quick plot of the shapefile
-#note: lwd sets the line width!
-plot(aoiBoundary,col="cyan1", border="black", lwd=3)
+This metadata are associated with all types of shapefiles (point, line,
+polygon).  The specific attributes may vary but all shapefiles will have
```

Comment:
This sentence is somewhat write but also potentially confusing . how about.

All shapefiles will have a unique ID attribute but other attributes may vary. 

or something that just calls attention to the fact that attributes aren't necessarily consistent.